source/storage.rst

```diff
@@ -0,0 +1,159 @@
+objects, which can have different types (e.g. files, images, documents) and
+are organized within containers (e.g. folders). Each object/file has is own
+URL, which can be used to access the resource, share the file with other
+people, setup custom metadata and access control lists. These objects are
+accessed and managed via a REST API (OpenStack SWIFT). Differently from the
+block storage, there is virtually no limit to the amount of data you can store,
+only the space used is accounted, you can access the data from any location
+(from any VM running at any EGI provider or even from other cloud providers
+or from your own laptop/browser), you can expose the data via external portals
+(using HTTP as transport protocols), you can set access control lists per
+container and even make the data publicly available. On the other hand, 
+data is accessed via a API requests, thus integration with existing applications
+may require a change to the application logic.
+
+A summary of the main differences between Block and Object Storage is reported
+in the following table.
+
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+|                    | Access                                 | Sharing      | Accounting           | Usage                           |
++====================+========================================+==============+======================+=================================+
+| **Block Storage**  | only from within a VM only at the same | Not possible | for the entire block | POSIX access, use as local disk |
+|                    | site the VM is located                 |              |                      |                                 |
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+| **Object Storage** | from any device connected to the       | Possible     | only for the data    | via HTTP requests to server     |
+|                    | internet                               |              | stored               |                                 |
++--------------------+----------------------------------------+--------------+----------------------+---------------------------------+
+
+According to your application needs, you may select one technology over the
+other. In general, block storage is a good and simple solution for temporary
+data and data which you do not need to share beside the single application
+running on a single VM. If you need to have your data exposed within portals
+or shared between different steps of your processing workflow, it is usually
+best to use the object storage.
+
+
+Block Storage
+-------------
+
+Block storage is created and managed via requests to specific APIs. Once the
+storage is attached to a VM, is managed as a regular block device that can be
```

Comment:
```suggestion
storage is attached to a VM, it is managed as a regular block device that can be
```

Reply:
I think that the term `block device` will probably be confusing/unclear lots of people